src/dioptra/restapi/queue/service.py

```diff
@@ -154,6 +154,26 @@ def get_all_unlocked(self, **kwargs) -> list[Queue]:
             .all()
         )
 
+    @staticmethod
```

Comment:
`staticmethod` can be removed, and `self` added to the `get_page` to conform to the recent changes for the Restapi.